src/core/atomicdex/models/qt.orderbook.model.cpp

```diff
@@ -374,13 +375,22 @@ namespace atomic_dex
         m_orders_id_registry.clear();
         for (auto&& order: m_model_data)
         {
+            // Maybe adding a suffix for segwit entries could avoid this?
+            // Working so far, but needs more testing to see if there are ripple effects due to the uuid suffix.
+            // Early tests confirm order selection and placing do not appear to be negatively affected.
```

Comment:
the effect i observed from this -segwit change is that you first see the -segwit entries in best-orders for couple seconds, then they are removed and only the non-segwit are left
i think it's good to only show one entry per coin, the duplicated entries with segwit were a bit annoying
problem is though that now people are not aware that they could also trade with segwit, saving some fees (though question is how much fees they actually save... i guess it makes a difference when using many inputs, but with a single input it will likely not make much of a difference)
this only applies to BTC, the other UTXO coins have such low fees that segwit is not needed for this
so i was thinking about this:
- make BTC-segwit default coin instead of BTC: https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/src/core/atomicdex/constants/dex.constants.hpp#L14 ... or add it next to BTC
- invert this filter for BTC and show the BTC-segwit entries instead of BTC in best-orders
what do you think about this?

Reply:
both the segwit and non segwit entries are exactly the same apart from ticker. The filtering is implied, I assume the orderbook model removes the nonstandard uuids. It could be a bit of work to get around this, so please add an issue for it. 
re: Default coins, I think having both would be good as it also lets the user know there is both available.

Reply:
resolved in https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/2280